README.md

```diff
@@ -71,7 +71,9 @@ $ wget https://github.com/meanjs/mean/archive/master.zip -O meanjs.zip; unzip me
 Don't forget to rename **mean-master** after your project name.
 
 ### Yo Generator
--Another way would be to use the [Official Yo Generator](http://meanjs.org/generator.html), which generates a copy of the MEAN.JS 0.3.x boilerplate and supplies multiple sub-generators to ease your daily development cycles. 
+Another way would be to use the [Official Yo Generator](http://meanjs.org/generator.html), which generates a copy of the MEAN.JS 0.4.x boilerplate and supplies an application generator to ease your daily development cycles. 
+
+Currently there are no sub generators for the current version of the MEAN.js project, but they are under development and will be coming soon.
```

Comment:
I see @lirantal point. That extra info might not be needed. It's sort of a moot point, in regards to it's relationship with this project. It seems that this type of statement is better suited for the generator Readme.